packages/amplify_auth_cognito/android/src/test/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AmplifyAuthCognitoPluginTest.kt

```diff
@@ -205,6 +208,23 @@ class AmplifyAuthCognitoPluginTest {
         verify(mockResult, times(1)).success(ArgumentMatchers.any<LinkedTreeMap<String, Any>>());
     }
 
+    @Test
+    fun signOut_signOutRequest() {
+        // Arrange
+        val data: HashMap<String, *> = hashMapOf(
+            "options" to hashMapOf(
+                    "globalSignOut" to true
+            )
+        )
+
+        // Act
+        val request = FlutterSignOutRequest(data)
+
+        // Assert
+        assert(request.signOutOptions is AuthSignOutOptions)
+        assert(request.signOutOptions.isGlobalSignOut)
```

Comment:
These look like the Kotlin stdlib [`assert`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/assert.html), instead of a jUnit `assertFoo`-style function. Was this your intention? Does the code base currently use `assert`?